src/lib/svg_text_utils.js

```diff
@@ -748,6 +748,7 @@ function alignHTMLWith(_base, container, options) {
         var x0 = getLeft() - cRect.left;
         var y0 = getTop() - cRect.top;
         var gd = options.gd || {};
+        gd._fullLayout._calcInverseTransform(gd);
```

Comment:
`gd` might be undefined considering above line
```js
var gd = options.gd || {};
```
If `gd` could be undefined there, then there is no `gd._fullLayout` and calling `gd._fullLayout._calcInverseTransform(gd);` could cause an error.
So here is my suggestion:
```suggestion
        if(gd._fullLayout && gd._fullLayout._calcInverseTransform) gd._fullLayout._calcInverseTransform(gd);
```

Reply:
Addressed in https://github.com/plotly/plotly.js/pull/5302/commits/3a5ca3a3281f1b4b0737c09da1ec2cfffa4b0396.